dask/dataframe/io/parquet.py

```diff
@@ -24,7 +24,7 @@
 
 
 def read_parquet(path, columns=None, filters=None, categories=None, index=None,
```

Comment:
In this case, might want to just pass **kwargs and refer the user to the documentation of the specific backend function.